yarn.lock

```diff
@@ -10867,6 +10867,16 @@ __metadata:
   languageName: node
   linkType: hard
 
+"typescript@npm:~5.2.2":
```

Comment:
no removal of the older version in lockfile?

Reply:
No. Since yarn modern patches all versions of TypeScript to support Plug'N'Play.